src/TCommandLine.cpp

```diff
@@ -105,6 +105,14 @@ bool TCommandLine::event( QEvent * event )
     if( event->type() == QEvent::KeyPress )
     {
         QKeyEvent *ke = static_cast<QKeyEvent *>( event );
+
+        // Shortcut for keypad keys
+        if(((int)ke->modifiers() & 0x20000000) && mpKeyUnit->processDataStream( ke->key(), (int)ke->modifiers() ) )
```

Comment:
Ok, I have made the modification you suggested.

This would allow users to override the enter key on the keypad if they wanted to.  I personally wouldn't see this as a problem because (a) there is another enter key which can't be overridden and (b) if the user decides they don't want to override the enter key on the keypad they can simply remove the binding or rebind it.  Please note that the proposed change would allow users to override enter key + modifiers, which could be useful.

An alternative would be to move this keypad check inside specific key cases, but this would mean a lot of duplicate code.

I personally have a mac and can test how the keys are handled but for the keyboard that most non-laptop mac users will have is the bluetooth one which has no keypad at all.  I do not have access to the wired full keyboard to test but I do have a USB full keyboard.  Though, for external 3rd party keyboards I'm not sure how predictable the handling is.